tests/deseq2/test_DESeq.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import patsy
+
+from inmoose.utils import Factor
+from inmoose.deseq2 import DESeq, makeExampleDESeqDataSet
+
+class Test(unittest.TestCase):
+
+    def test_DESeq(self):
```

Comment:
Shouldn't we also test that deseq give the good results?

Reply:
Agreed. So far the tests replicate those on the original R package.
We can make them more exhaustive.